content/docs/command-reference/checkout.md

```diff
@@ -10,39 +10,39 @@ usage: dvc checkout [-h] [-q | -v] [--summary] [-d] [-R] [-f]
                     [--relink] [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these stages or .dvc files.
-                 Using -R, directories to search for stages or .dvc
-                 files can also be given.
+  targets       Limit command scope to these tracked files/directories,
+                .dvc files, or stage names.
 ```
 
 ## Description
 
-`.dvc` and `dvc.lock` [files](/doc/user-guide/dvc-files-and-directories) act as
-pointers to specific version of data files or directories tracked by DVC. This
-command synchronizes the workspace data with the versions specified in the
-current `.dvc` and `dvc.lock` files.
+This command is usually needed after `git checkout`, `git clone`, or any other
+operation that changes the `dvc.lock` or `.dvc` in the workspace. It restores
```

Comment:
if `dvc.yaml` is the source of truth for checkout then we should mention it as well probably .. or come up with some way to mention them at once like abbr DVC files abbr.

Reply:
Agree... But this one is pretty tricky. Mentioned dvc.yaml as a note for now in a919f85f.

Reply:
> or come up with some way to mention them at once like abbr DVC files abbr

I think we definitely need to do this and not just here but in many documents... But I'm going to extract it: see #1663